tests/rptest/tests/cluster_config_test.py

```diff
@@ -69,6 +118,14 @@ def test_get_config(self, legacy):
         # Some arbitrary property to check syntax of result
         assert 'kafka_api' in node_config
 
+    @cluster(num_nodes=1)
+    def test_upgrade_from_redpanda_yaml(self):
+        """
+        Verify that on first start, values are imported from redpanda.yaml
+        to facilitate upgrades, but on subsequent startups we just emit
+        a log message to say we're ignoring them.
+        """
```

Comment:
did the docstring from this empty test belong in the non-empty test above ` test_upgrade_redpanda_yaml` ?